en/docs/learn/consume-api/invoke-apis/invoke-apis-using-tools/try-out-using-postman.md

```diff
@@ -0,0 +1,57 @@
+# Try out using Postman.
+
+Let`s see how to download a postman collection.
+
+1.  Sign in to the WSO2 Developer Portal (`https://<hostname>:9443/devportal`) and click an API (e.g. `Pizza` ) to go to the API overview.
+
+    ![({{base_path}}/assets/attachments/postman/postman_api_overview.png)]({{base_path}}/assets/attachments/postman/postman_api_overview.png)
+
+
+2.  Click `Try Out`to go to the Try out section.
+
+    ![({{base_path}}/assets/attachments/postman/postman_try_out.png)]({{base_path}}/assets/attachments/postman/postman_try_out.png)
+
+3.  If you haven`t subscribed to this API, you should [Subscribe to an API]({{base_path}}/learn/consume-api/manage-subscription/subscribe-to-an-api).
+
+    Then choose the application name and click the `postman collection` button to download the postman collection.
+
+    ![({{base_path}}/assets/attachments/postman/postman_download_collection.png)]({{base_path}}/assets/attachments/postman/postman_download_collection.png)
+    
+## Try out collection in postman.
+
+
+1.  As there's a security for the postman collection, we have to get the authentication code. First go to `Subscriptions`.
```

Comment:
Do not use the term "we" in the docs. Either use the term "you" or if we are talking from WSO2's point of view, say "WSO2". Example, WSO2 recommends that you enable the SSL connection in a production environment.